src/Dependencies/.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*.cs]
+
+# IDE0240: Remove redundant nullable directive
+# The directive needs to be included since all sources in a source package are considered generated code
+# when referenced from a project via package reference.
+dotnet_diagnostic.IDE0240.severity = none
```

Comment:
It would be good if we could actually enforce somehow that a `#nullable` directive is present in source package files, so we don't forget when new files are added in the future. I'm not sure how to do that though.

Reply:
I think we would need an analyzer that understands source packages.